### ISSUE TITLE
[Light box media]: QA: Extra unwanted tab focus is observed in the end of the page while reverse navigation

### ISSUE THREAD

**Issue by gopinathan-n** [carbon-design-system/carbon-for-ibm-dotcom#9390]
### Description

Page Element:
Extra focus in the end of the page

Issue Description:
Extra unwanted tab focus is observed in the end of the page while reverse navigation.
Note: This is happening only in the reverse navigation.

Expected:
Unwanted tab focus should be removed.

User Impact:
When standard methods of keyboard navigation is not supported, this impact the user ability to interact with the page controls. This impacts Keyboard only users/Screen reader users

### Component(s) impacted

Light box media viewer

### Browser

Firefox

### Carbon for IBM.com version

v1.38.0-rc.0

### Severity

Severity 4 = The problem is not visible to or noticeable to an average user. Affects minor functionality, no workaround needed.

### Application/website

Carbon for IBM.com

### Package

@carbon/ibmdotcom-web-components

### CodeSandbox example

https://carbon-design-system.github.io/carbon-for-ibm-dotcom/canary/web-components/

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/blob/main/.github/CODE_OF_CONDUCT.md)
- [ ] I checked the [current issues](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/issues) for duplicate issues

**Comment by andy-blum** [comment]
Can you provide a direct link to an example? Looking at the canary storybook pages for the lightbox media, I can't replicate this issue.
